devops/deploy.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+ENV_FILE=.cred.env
+DOCKER_FOLDER=docker
+DOCKER_COMPOSE=docker-compose.yml
+CYCLER_DOCKERFILE=Dockerfile.cycler
+DB_SYNC_DOCKERFILE=Dockerfile.db_sync
+INT_RE='^[0-9]+$'
+
+arg1=$1
```

Comment:
Remove it. Before using an argument, you must check the number of arguments

Reply:
It is used to store the passed arguments and difference them from the arguments passed to internal functions. Rewritten using CAPS to represent constant value (if no argument passed, the value is empty).